### PR TITLE
Fix image locations

### PR DESCRIPTION
**Pull request by RobbeSneyders**
These image locations were fixed in the notebook, but still outdated in `pipeline.py`

### CODE REVIEW

File: src/pipeline.py [hunk 1]
```diff
@@ -47,16 +47,16 @@
...
     arguments={
         "retries": 2,
         "min_image_size": 0,
     },
 )
 
-resize_images = ComponentOp.from_registry(
-    name="resize_images",
+resize_images = ComponentOp(
+    component_dir="components/resize_images",
```
Comment (mrchtr) [inline src/pipeline.py:59]: The folder dosen't exist in the repository.

**Review by mrchtr — COMMENTED** [review]
Thanks @RobbeSneyders. I think something gone wrong in your last PR #5.
The notebook is gone and the resize image component is missing.

**Comment by RobbeSneyders** [comment]
Good catch. Rookie mistake :sweat_smile: 
Added them back to this PR.

**Review by mrchtr — APPROVED** [review]
Thanks @RobbeSneyders! Everything is back again :)

**Pull request merged**